packages/jest-repl/src/cli/repl.ts

```diff
@@ -5,30 +5,34 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @flow
  */
 
-import type {GlobalConfig, ProjectConfig} from 'types/Config';
-
-declare var jestGlobalConfig: GlobalConfig;
-declare var jestProjectConfig: ProjectConfig;
-declare var jest: Object;
+declare var jestGlobalConfig: Config.GlobalConfig;
```

Comment:
why `declare var`?

Reply:
Oh that was there from the beginning. I was unsure of what that was supposed to do since those variables are used down in the code. Let me see what removing them does